### ISSUE TITLE
Feature Request: Close/minimize to system tray (Windows)

### ISSUE THREAD

**Issue by nathlrowe** [ProtonMail/inbox-desktop#19]
Add option to have application be minimized to the system tray instead of quitting completely.

My ideal is to be able to have a lightweight Proton Mail process running in the background, with an icon in the system tray, giving me notifications when a new email arrives or on calendar events. For example, I have my Discord app configured to minimize to tray when I click "X", instead of closing completely. This clears space on my taskbar, while keeping me notified of Discord messages.

I would also want "Start Proton Mail on startup" and "Start Proton Mail minimized to tray" options as well.

**Comment by Zie0** [comment]
I believe this issue is a duplicate of https://github.com/ProtonMail/inbox-desktop/issues/10

**Comment by Salieri-Studio** [comment]
I don't think that it's a duplicate. The issue you have mentioned is for Linux desktop environment, and this issue was opened for Windows desktop version.> I believe this issue is a duplicate of https://github.com/ProtonMail/inbox-desktop/issues/10
